app/lib/service/announcement/backend.dart

```diff
@@ -27,6 +27,8 @@ class AnnouncementBackend {
     final value = await secretBackend.lookup(SecretKey.announcement);
     if (value != null && value.trim().isNotEmpty) {
```

Comment:
Nit: Perhaps switch the branches for slightly simpler control flow?

You could maybe also skip the `value.trim().isNotEmpty` check. Why can't we set the announcement  to an empty string?

Reply:
We have a CLI utility that sets the Datastore value, but nothing that deletes it. If it is set to `''` (empty string), it is assumed the same as being deleted.